temp/legacy-code/src/main/java/com/ivy/legacy/ui/component/transaction/TransactionCard.kt

```diff
@@ -310,13 +314,13 @@ private fun TransactionHeaderRow(
             )
         }
     } else {
-        Row(
+        FlowRow(
+            horizontalArrangement = Arrangement.spacedBy(12.dp),
             modifier = Modifier.padding(horizontal = 20.dp),
-            verticalAlignment = Alignment.CenterVertically
+            verticalArrangement = Arrangement.spacedBy(8.dp)
         ) {
```

Comment:
The FlowRow looks good, how the items rendee when they are on a single line (most cases)? They must be vertically centered.

When you update the Paparazzi screenshots and commit them, we'll see that case hopefully.

Reply:
Which reminds, please add a new Paparazzi test with the account going on a separate row

Reply:
ok i understand why paparazzi tests not passed, it is due to something that I've never modified. Now, I have to follow the guide in "https://github.com/Ivy-Apps/ivy-wallet/blob/main/docs/CI-Troubleshooting.md#paparazzi-tests" and update each single screen?
<img width="828" alt="Screenshot 2024-06-21 alle 00 48 43" src="https://github.com/Ivy-Apps/ivy-wallet/assets/78317934/7fc4b3a2-d7b1-4b26-8e7c-30ae3fa78a45">
This is an example of a UI that has been modified

Reply:
@giuseppe2028 you modified the transaction and it's used on many screens. You can run:
```
./gradlew recPaD
```

to update all screens that have changed at once